config/v1/types_cluster_version.go

```diff
@@ -399,6 +399,15 @@ const (
 	// Managers deployed on top of OpenShift. They help you to work with cloud
 	// provider API and embeds cloud-specific control logic.
 	ClusterVersionCapabilityCloudControllerManager ClusterVersionCapability = "CloudControllerManager"
+
+	// ClusterVersionCapabilityOptionalMonitoring manages the cluster monitoring stack which is responsible for gathering and
+	// processing metrics from the in-house and user workloads. The following CRDs are constitute this capability:
+	// - TODO
```

Comment:
Presumably we should not add this capability until we know the full scope of what this capability will and will not do?